main.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"log"
+	"os"
+	"os/signal"
+	"time"
+
+	"github.com/Skycoin/git-telegram-bot/pkg/githandler"
+	"github.com/Skycoin/git-twitter-bot/internal/config"
+	"github.com/dghubble/go-twitter/twitter"
+	"github.com/dghubble/oauth1"
+)
+
+func main() {
+	l := log.New(os.Stdout, "git-twitter-bot", log.LstdFlags)
+
+	cfg, err := config.NewBotConfig()
+	if err != nil {
+		l.Fatal(err)
+	}
+	oauthCfg := oauth1.NewConfig(cfg.ConsumerKey, cfg.ConsumerSecret)
+	oauthToken := oauth1.NewToken(cfg.AccessToken, cfg.AccessSecret)
+	hc := oauthCfg.Client(oauth1.NoContext, oauthToken)
```

Comment:
Good job, but I think it's better to use descriptive and unambiguous names. Maybe for now some of them (same as **tc** [Twitter client] or **l** [log]) are clear and some of them are not (same as **hc**), but in the future and when the project grows up it could be a problem.

Reply:
Cool, thanks. yeah will use more descriptive names.